documentation/index.md

```diff
@@ -8,89 +8,129 @@ sd_hide_title: true
 :alt: Docs Header Image
 :width: 1104
 ```
+#
+
+
+{.frontpage-heading1}
+# Asset Management
```

Comment:
In my opinion these h1 headings are way too big for this purpose. I think the h3 size (1.5em / 24px) would be better.

Reply:
fixed